composer.json

```diff
@@ -12,14 +12,14 @@
         { "name": "Jeremy Mikola", "email": "jmikola@gmail.com" }
     ],
     "require": {
-        "php": "^7.1",
-        "symfony/framework-bundle": "^3.4|^4",
-        "symfony/console": "^3.4|^4",
-        "symfony/dependency-injection": "^3.4|^4",
-        "symfony/property-access": "^3.4|^4",
-        "pagerfanta/pagerfanta": "^1.0.5|^2.0",
+        "php": "^7.2",
```

Comment:
I suggest to bump PHP version to 7.2 as 7.1 end of security support is arriving next month.

Reply:
This will also allow to use the `object` type-hint in method/function signatures

Reply:
:+1: